R/eval.R

```diff
@@ -37,27 +52,67 @@ rust_eval <- function(code, env = parent.frame(), ...) {
     ui_throw("decoy function; should never be called.")
   }
 
+  snippet_hash <- rlang::hash(list(the$count, code))
```

Comment:
Could you add a brief comment here explaining why the hash? You give that explanation in the body of the PR, but I think it's good to record it in the code itself also.

Reply:
Done